README.md

```diff
@@ -1 +1,9 @@
-# SkyNest
\ No newline at end of file
+# SkyNest
+The Sky Nest cloud service is a scalable and safe system designed to provide data storage and user control.
+
+## Frontend
+
+## Backend
+Run backend server/databases using <code>docker compose up</code> in the root folder. A <code>.env</code> should also be in root. 
+
```

Comment:
Good practice is to comment the whole process how the app is starting (for new colleagues, FE/QA team). For example add prerequisites section like Java 11, Docker etc etc.. How to run locally in IntelliJ (with all configurations , commands or what ever is needed) or via terminal if needed, what needs to be started before (like databases, commands like you already have done )

Reply:
Ok, I'll expand the readme.